app/models/image.rb

```diff
@@ -0,0 +1,10 @@
+class Image
+  include ActiveModel::Serialization
+
```

Comment:
I'm a little unsure about this behaviour. You're claiming that this is an 'image' object, but its ID field is that of a 'person' object. Doesn't feel quite right?

Reply:
I see you point, but the initialize helps to understand what actually the id is:

```
def initialize(person_id, image_url)
    @id = person_id
    @url = image_url
  end
```

Image is an entity per se, so it's fine to have it as a model imo,
the models folders do not necessary need to contain only record model, there are other classes not activerecord.